pkg/engine/files.go

```diff
@@ -156,5 +156,5 @@ func (f files) Lines(path string) []string {
 		return []string{}
 	}
 
-	return strings.Split(string(f[path]), "\n")
+	return strings.Split(strings.TrimRight(string(f[path]), "\n"), "\n")
```

Comment:
On another note, does this work on Windows? (I didn't read any details about how we load the files here)

Reply:
If it doesn't, it never has. CRLF files would leave the CR at the end. Changing that behavior is beyond the scope of this PR.